log4j2-elasticsearch-core/src/main/java/org/appenders/log4j2/elasticsearch/ElasticsearchAppender.java

```diff
@@ -73,7 +75,7 @@ public static Builder newBuilder() {
         return new Builder();
     }
 
-    public static class Builder implements org.apache.logging.log4j.core.util.Builder<ElasticsearchAppender>  {
+    public static class Builder extends AbstractFilterable.Builder implements org.apache.logging.log4j.core.util.Builder<ElasticsearchAppender>  {
```

Comment:
I'm not a fan of extending `AbstractFiterable.Builder` here. `AbstractAppender.Builder` looks more appropriate, but let's defer this decision now.

Reply:
Yes, looks better.